src/main/java/rx/internal/operators/OperatorObserveOn.java

```diff
@@ -105,12 +109,15 @@ public ObserveOnSubscriber(Scheduler scheduler, Subscriber<? super T> child, boo
             this.recursiveScheduler = scheduler.createWorker();
             this.delayError = delayError;
             this.on = NotificationLite.instance();
-            this.bufferSize = (bufferSize > 0) ? bufferSize : RxRingBuffer.SIZE;
+            int calculatedSize = (bufferSize > 0) ? bufferSize : RxRingBuffer.SIZE;
+            this.limit = calculatedSize - (calculatedSize >> 2);
```

Comment:
What do you think about a less efficient (but more readable): `(int) (calculatedSize * 0.75)` ?